Sources/StreamChat/Repositories/AuthenticationRepository.swift

```diff
@@ -328,7 +328,7 @@ class AuthenticationRepository {
             // to `disconnected` when environment was prepared correctly
             // (e.g. current user session is successfully restored).
             connectionRepository?.forceConnectionStatusForInactiveModeIfNeeded()
-            connectionRepository?.connect(userInfo: userInfo, completion: onCompletion)
```

Comment:
Why don't we need this anymore? 🤔 Was this an issue before?

Reply:
`userInfo` was not used in `connect`